service/measurement-server/README.md

```diff
@@ -0,0 +1,127 @@
+# Service: CCNP Measurement Server
+
+Measurements are key assets in the confidential computing world for user to verify and determine the trustworthiness of the environment and its underlying platform.
+Users can utilize different kinds of measurements to monitor or validate the operations performed within the platform.
+
+This service (which is actually a GRPC server) will provide support to fetch the measurements for confidential cloud native environments, including TDX RTMR measurements, TPM PCR measurements and TEE reports.
+
+
+
+## Introduction
+
+This service provides functionality to fetch measurements for confidential cloud native environments, both platform level(`PAAS` option) and container level (`SAAS` option). Using this service, user can fetch different
```

Comment:
Suggest to use "PaaS" and "SaaS"

Reply:
These two are using the same capital words as protobuf defined. May i still use the capital ones?

Reply:
sure, not a big deal